akka-management/src/main/scala/com/lightbend/rp/akkamanagement/AkkaManagement.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2017 Lightbend, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.lightbend.rp.akkamanagement
+
+import akka.actor._
+import akka.management.{ AkkaManagement => AkkaMgmt }
+import com.lightbend.rp.common.{ AkkaManagementModule, Module, Platform }
+
+final class AkkaManagement(system: ExtendedActorSystem) extends Extension {
```

Comment:
Call it AkkaManagementAutostart to avoid confusion?

Reply:
:+1: thanks, this is done.